rails/app/models/external_report.rb

```diff
@@ -35,6 +35,10 @@ def url_for_offering(offering, user, protocol, host, additional_params = {})
       params = offering_report_params(offering, grant, user, url_options, additional_params)
     end
 
+    if offering.runnable.logging
+      params[:logging] = 'true'
+    end
+
```

Comment:
I see you are using the clazz.logging property down below. I think the logic for external activity launches is that if the clazz.logging is enabled then the logging param is sent to all external activities launched from the class. So to follow that pattern the conditional above should be something like `offering.runnable.logging || offering.clazz.logging`